src/App/routes.tsx

```diff
@@ -1101,6 +1101,36 @@ const fieldReportFormNew = customWrapRoute({
     },
 });
 
+const fieldReportsFormDetails = customWrapRoute({
+    path: 'field-reports/:fieldReportId',
+    component: {
+        render: () => import('#views/FieldReportDetails'),
+        props: {},
+    },
+    parent: root,
+    wrapperComponent: Auth,
+    context: {
+        title: 'All Field Reports',
+        visibility: 'anything',
+    },
+});
+
+//  FIX ME: Add field report form link
+
```

Comment:
Let's remove this. 
Note: it should be // FIXME:

Reply:
Field report form in not available